Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -597,13 +597,11 @@ extension RemoteDataServicesManager {
 
 extension RemoteDataServicesManager {
 
```

Comment:
Getting rid of validatePushNotificationSource as mentioned above. 

The method serviceForPushNotification will find the appropriate service for a push notification. We will use that service then to ask for a RemoteCommand from the push notification.